db/seeds.rb

```diff
@@ -5,3 +5,21 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+ Shelter.destroy_all
```

Comment:
Make sure you check out `dependent: :destroy` or only delete the child objects (like pets) or join object (application_pets) first before destroying the parent objects. Otherwise, you will get errors when running seeds locally and when deploying!